canonical-web-and-design/snapcraft.io
# Convert brand store selectors to typescript

## Done
- Convert brand store selectors to TypeScript
- Convert brand store Redux store to TypeScript

## QA
- Go to https://snapcraft-io-3786.demos.haus/admin/test1LaNg1xi6eonae5R/settings
- Check it works as expected
- Go to https://snapcraft-io-3786.demos.haus/admin/test1LaNg1xi6eonae5R/members
- Check it works as expected

## Issue
Fixes #3773, #3772
